tests/samples/rename_git.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/added b/moved
+similarity index 85%
+rename from added
+rename to moved
+index a071991..4dbab21 100644
+--- a/added
++++ b/moved
```

Comment:
Sure. fwiw I do not need it anymore as I managed to not need unidiff but only use GitPython.